cmd/kperf.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/zhanggbj/kperf/core"
+)
+
+func main() {
+	if err := core.NewPerfCommand().Execute(); err != nil {
+		fmt.Println("failed to execute kperf command:", err)
```

Comment:
Better to change all the `fmt.Print` to `log.Print`

Reply:
Fixed